package.json

```diff
@@ -30,17 +30,21 @@
     "@fluentui/react": "^8.52.3",
     "core-js": "^3.9.1",
     "es6-promise": "^4.2.8",
+    "eslint": "^7.32.0",
```

Comment:
I don't think we need to install eslint. Office introduces eslint and prettier through its boilerplate code

Reply:
I guess it depends on IDE, the people that developed this parts used jetbrains ide.

Reply:
I can remove and try without these npm packages.

Reply:
Did you try removing Eslint and tested without it?

Reply:
I think i need to remove all changes in package.json and build towards a working version for node 14. I did not think 14 was still used here.